### PR TITLE
Added modules for login and signup

### PR DESCRIPTION
**Pull request by raynelle03**

### CODE REVIEW

File: iosHackthonApp/Modules/Features/Login/.gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.DS_Store
+/.build
+/Packages
```
Comment (Gmiller290488) [inline iosHackthonApp/Modules/Features/Login/.gitignore]: Sorry, I should have said earlier - you can delete this file

File: iosHackthonApp/Modules/Features/Login/Sources/Login/LoginView.swift [hunk 2]
```diff
@@ -27,7 +28,7 @@ struct LoginView: View {
 		UINavigationBar.appearance().scrollEdgeAppearance = coloredNavAppearance
 	}
 
-	var body: some View {
+public var body: some View {
```
Comment (Gmiller290488) [inline iosHackthonApp/Modules/Features/Login/Sources/Login/LoginView.swift]: can we indent this whole body please

File: iosHackthonApp/Modules/Features/SignUp/.gitignore [hunk 3]
```diff
@@ -0,0 +1,5 @@
+.DS_Store
```
Comment (Gmiller290488) [inline iosHackthonApp/Modules/Features/SignUp/.gitignore]: Same with this file - safe to delete

File: iosHackthonApp/Modules/Features/SignUp/Sources/SignUp/SignUpView.swift [hunk 4]
```diff
@@ -11,9 +11,13 @@ import Strings
 import Theming
 import Components
 
-struct SignUpView: View {
+public struct SignUpView: View {
 
-	var body: some View {
+	public init() {
+
+	}
```
Comment (Gmiller290488) [inline iosHackthonApp/Modules/Features/SignUp/Sources/SignUp/SignUpView.swift]: Can we move the } of this init up to the same line

File: iosHackthonApp/Modules/Features/SignUp/Sources/SignUp/SignUpView.swift [hunk 5]
```diff
@@ -11,9 +11,13 @@ import Strings
...
+
+public var body: some View {
```
Comment (Gmiller290488) [inline iosHackthonApp/Modules/Features/SignUp/Sources/SignUp/SignUpView.swift]: indentation please

File: iosHackthonApp/Modules/Features/Login/.gitignore (hunk 1 above)
Comment (raynelle03) [reply iosHackthonApp/Modules/Features/Login/.gitignore]: Done

File: iosHackthonApp/Modules/Features/Login/Sources/Login/LoginView.swift (hunk 2 above)
Comment (raynelle03) [reply iosHackthonApp/Modules/Features/Login/Sources/Login/LoginView.swift]: Yup..done

File: iosHackthonApp/Modules/Features/SignUp/.gitignore (hunk 3 above)
Comment (raynelle03) [reply iosHackthonApp/Modules/Features/SignUp/.gitignore]: yes deleted

File: iosHackthonApp/Modules/Features/SignUp/Sources/SignUp/SignUpView.swift (hunk 4 above)
Comment (raynelle03) [reply iosHackthonApp/Modules/Features/SignUp/Sources/SignUp/SignUpView.swift]: Done

File: iosHackthonApp/Modules/Features/SignUp/Sources/SignUp/SignUpView.swift (hunk 5 above)
Comment (raynelle03) [reply iosHackthonApp/Modules/Features/SignUp/Sources/SignUp/SignUpView.swift]: Done

**Review by Gmiller290488 — APPROVED** [review]
Nice one 👏

**Review by kirstybutler — APPROVED** [review]

**Pull request merged**